src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the `ExcavatorProducer` class.